.github/workflows/pr-view.yml

```diff
@@ -0,0 +1,221 @@
+                - value: '{{ userdata.count("Azure/functions-container-action","idle") }}'
+                  url: '{{ userdata.url("Azure/functions-container-action","idle") }}'
+                - value: '{{ userdata.count("Azure/functions-container-action","closed") }}'
+                  url: '{{ userdata.url("Azure/functions-container-action","closed") }}'
+              - - value: 'mysql' 
+                - value: '{{ userdata.count("Azure/mysql","") }}'
+                  url: '{{ userdata.url("Azure/mysql","") }}'
+                - value: '{{ userdata.count("Azure/mysql","stale") }}'
+                  url: '{{ userdata.url("Azure/mysql","stale") }}'
+                - value: '{{ userdata.count("Azure/mysql","idle") }}'
+                  url: '{{ userdata.url("Azure/mysql","idle") }}'
+                - value: '{{ userdata.count("Azure/mysql","closed") }}'
+                  url: '{{ userdata.url("Azure/mysql","closed") }}'
+              - - value: 'postgresql' 
+                - value: '{{ userdata.count("Azure/postgresql","") }}'
+                  url: '{{ userdata.url("Azure/postgresql","") }}'
+                - value: '{{ userdata.count("Azure/postgresql","stale") }}'
+                  url: '{{ userdata.url("Azure/postgresql","stale") }}'
+                - value: '{{ userdata.count("Azure/postgresql","idle") }}'
+                  url: '{{ userdata.url("Azure/postgresql","idle") }}'
+                - value: '{{ userdata.count("Azure/postgresql","closed") }}'
+                  url: '{{ userdata.url("Azure/postgresql","closed") }}'
+              - - value: 'powershell' 
+                - value: '{{ userdata.count("Azure/powershell","") }}'
+                  url: '{{ userdata.url("Azure/powershell","") }}'
+                - value: '{{ userdata.count("Azure/powershell","stale") }}'
+                  url: '{{ userdata.url("Azure/powershell","stale") }}'
+                - value: '{{ userdata.count("Azure/powershell","idle") }}'
+                  url: '{{ userdata.url("Azure/powershell","idle") }}'
+                - value: '{{ userdata.count("Azure/powershell","closed") }}'
+                  url: '{{ userdata.url("Azure/powershell","closed") }}'
+              - - value: 'sql-action ' 
+                - value: '{{ userdata.count("Azure/sql-action","") }}'
+                  url: '{{ userdata.url("Azure/sql-action","") }}'
+                - value: '{{ userdata.count("Azure/sql-action","stale") }}'
+                  url: '{{ userdata.url("Azure/sql-action","stale") }}'
+                - value: '{{ userdata.count("Azure/sql-action","idle") }}'
+                  url: '{{ userdata.url("Azure/sql-action","idle") }}'
+                - value: '{{ userdata.count("Azure/sql-action","closed") }}'
+                  url: '{{ userdata.url("Azure/sql-action","closed") }}'
+              
+          
+                    
+        token: ${{ github.token }}
+
+    - name: Commit files
+      id: commit
+      run: |
```

Comment:
add continue-on-error : true
and update config as well :)

Reply:
done.